<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1465</width>
    <height>752</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>.SF NS Text</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Lab 05</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="add_dot_label">
        <property name="font">
         <font>
          <family>.SF NS Text</family>
          <pointsize>18</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Добавление точки</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="x_label">
          <property name="text">
           <string>X</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="x_spinbox"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="y_label">
          <property name="text">
           <string>Y</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="y_spinbox"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="add_dot_btn">
        <property name="text">
         <string>Добавить точку</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="close_figure_btn">
        <property name="text">
         <string>Замкнуть фигуру</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="color_chooser">
          <property name="text">
           <string>Выбор цвета</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGraphicsView" name="color_viewer">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="paint_figure_btn">
        <property name="text">
         <string>Закрасить фигуру</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clean_screen_btn">
        <property name="text">
         <string>Очистить холст</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="do_slow_drowing">
        <property name="text">
         <string>Задержка закраски</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="coord_table">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <column>
         <property name="text">
          <string>X</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Y</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGraphicsView" name="main_scene"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1465</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuinfo">
    <property name="title">
     <string>info</string>
    </property>
    <addaction name="actionabout"/>
   </widget>
   <addaction name="menuinfo"/>
  </widget>
  <action name="actionabout">
   <property name="text">
    <string>about</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
